src/components/super/i-static-page/README.md

```diff
@@ -260,6 +260,7 @@ const deps = {
  *   1. `lib` - the external library, i.e, something from `node_modules`
  *   2. `src` - the internal resource, i.e, something that builds from the `/src` folder
  *   3. `output` - the output library, i.e, something that builds to the `/dist/client` folder
+ *   4. `external` - externally hosted library, i.e. from a CDN
  *
  * @typedef {('lib'|'src'|'output')}
```

Comment:
Shouldn't the new value be added to the type definition as well?

Reply:
Updated it, thanks!